fixed_array.h

```diff
@@ -47,11 +46,14 @@ class fixed_array
 		RDE_ASSERT(i < size());
 		return m_data[i];
 	}
-
-private:
-	T	m_data[N];
 };
 
+#if RDE_HAS_CPP11
+// A condensed alias using standard container nomenclature
+template<typename T, size_t N>
+using array = fixed_array<T, N>;

```

Comment:
I would be in favor of adopting `array<T,N>` as the actual name, given the fact that fixed allocation is a fundamental requirement for an array to begin with, along with being homogeneous, contiguous, etc. Your thoughts @msinilo?